tripal_chado/tripal_chado.install

```diff
@@ -575,3 +575,44 @@ function tripal_chado_update_10406() {
   $upgradable_properties = ['linker_synonym_fkey_id'];
   tripal_chado_field_reload($upgradable_types, $upgradable_properties, TRUE);
 }
+
+/**
+ * Adds third party settings for content types that have multiple
+ * content types derived from one Chado base table. PR#1991
+ */
+function tripal_chado_update_10407() {
+  $settings = [
+    // analysis table
+    'genome_assembly' => 'analysisprop',
+    'genome_annotation' => 'analysisprop',
+    // feature table
+    'gene' => 'feature',
+    'mrna' => 'feature',
+    'QTL' => 'feature',
+    'sequence_variant' => 'feature',
+    'genetic_marker' => 'feature',
+    'phenotypic_marker' => 'feature',
+    // featuremap table
+    'genetic_map' => 'featuremapprop',
+    'physical_map' => 'featuremapprop',
+    // project table
+    'genome_project' => 'projectprop',
+    // stock table
+    'germplasm' => 'stock',
+    'breeding_cross' => 'stock',
+    'germplasm_variety' => 'stock',
+    'ril' => 'stock',
+  ];
```

Comment:
What about Tripal 4 sites that have their own custom content types already? They will not get touched by this update hook.

Reply:
> What about Tripal 4 sites that have their own custom content types already? They will not get touched by this update hook.

The update hook is now generic - it uses the "fixed_type" settings.

Reply:
Confirmed resolved in new update hook.